src/Inspetor/Model/InspetorItem.php

```diff
@@ -78,7 +78,7 @@ public function isValid() {
             throw new InspetorItemException(7002);
         }
 
-		if (!$this->session_id) {
+		if (!$this->session_ids || !is_array($this->session_ids) || empty($this->session_ids)) {
             throw new InspetorItemException(7003);
```

Comment:
Why does this check raise [exception 3](https://github.com/inspetor/inspetor-php/pull/31/files#diff-4473d201e86832c13eb15c239be3b044R30)? Shouldn't it raise either 6 or 10?

Reply:
Ah, that's an `InspetorEventException` you've linked to there. I believe this is correctly triggering `InspetorItemException` 7003: https://github.com/inspetor/inspetor-php/pull/31/files#diff-529d153d857c56070376f00c26497b7eR30

Reply:
oops, my bad!